packages/cli/src/commands/j1Export.ts

```diff
@@ -23,6 +24,10 @@ export function j1Export() {
       'The directory where entities/relationships will be downloaded',
       DEFAULT_EXPORT_DIRECTORY,
     )
+    .option(
+      '--account <account>',
+      'The JupiterOne account you are importing entities/relationships into',
```

Comment:
This is `j1Export`, seems this should talk in terms of exporting, not importing.